### PR TITLE
Add specification for cl_ext_image_from_buffer

### PR DESCRIPTION
**Pull request by kpet**
Change-Id: Ieca35aedd775379310417b94387ea0776c4a0d55
Signed-off-by: Kevin Petit <kevin.petit@arm.com>

### CODE REVIEW

**Comment by bashbaug** [comment]
If this one is ready we should probably merge it too, or at least the XML changes, since the enums are in the upstream headers (https://github.com/KhronosGroup/OpenCL-Headers/pull/196).

File: extensions/cl_ext_image_from_buffer.asciidoc [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+Alastair Murray, Codeplay Software Ltd. +
+Balaji Calidas, Qualcomm +
+
+== Notice
+
+include::../copyrights.txt[]
+
+== Status
+
+Draft spec, NOT APPROVED!!
```
Comment (bashbaug) [inline extensions/cl_ext_image_from_buffer.asciidoc:39]: Consider updating this status.

**Review by bashbaug — APPROVED** [review]

**Comment by bashbaug** [comment]
Is this ready to go?  Needs a trivial rebase if so.

**Comment by kpet** [comment]
@bashbaug Now rebased (and updated the status and copyright dates). Should be ready to go.

**Pull request merged**